src/gulpfile.babel.js

```diff
@@ -59,8 +59,8 @@ const wpPot = require( 'gulp-wp-pot' ); // For generating the .pot file.
 const sort = require( 'gulp-sort' ); // Recommended to prevent unnecessary changes in pot-file.
 const cache = require( 'gulp-cache' ); // Cache files in stream for later use.
 const remember = require( 'gulp-remember' ); //  Adds all the files it has ever seen back into the stream.
-const plumber = require( 'gulp-plumber' ); // Prevent pipe breaking caused by errors from gulp plugins.
```

Comment:
Does `pump` handle preventions of pipe breaking caused by errors from gulp plugins?

Reply:
I don't think `plumber` can be replaced by `pump`. A mistake in Sass file caused the process to break down without `plumber` into this error.

```sh
[20:04:28] Error in plugin "gulp-sass"
Message:
    assets/css/style.scss
Error: Invalid CSS after "asdasd": expected "{", was ""
        on line 23 of assets/css/style.scss
>>
   ^

Details:
    status: 1
    file: /Users/ahmadawais/Documents/Web/Dev/Local/local/app/public/wp-content/themes/any-theme/assets/css/style.scss
    line: 23
    column: 1
    formatted: Error: Invalid CSS after "asdasd": expected "{", was ""
        on line 23 of assets/css/style.scss
>>
   ^

    messageFormatted: assets/css/style.scss
Error: Invalid CSS after "asdasd": expected "{", was ""
        on line 23 of assets/css/style.scss
>>
   ^

    messageOriginal: Invalid CSS after "asdasd": expected "{", was ""
    relativePath: assets/css/style.scss

[20:04:28] '<anonymous>' errored after 35 s
[20:04:28] 'default' errored after 35 s
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! your-project@1.0.0 start: `gulp`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the your-project@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ahmadawais/.npm/_logs/2018-09-14T15_04_28_266Z-debug.log
```

With `plumber` the stream stays active and waits for me to correct the mistake.

```sh
[20:09:48] Starting 'styles'...
Error in plugin "sass"
Message:
    assets/css/style.scss
Error: Invalid CSS after "asdsasdasd": expected "{", was ""
        on line 23 of assets/css/style.scss
>>
   ^

[20:09:48] gulp-notify: [Error running Gulp]

❌  ===> ERROR: assets/css/style.scss
Error: Invalid CSS after "asdsasdasd": expected "{", was ""
        on line 23 of assets/css/style.scss
>>
   ^


[20:09:48] Finished 'styles' after 19 ms
```

![image](https://on.ahmda.ws/2b599445152c/c)

Also, as soon as I fix the mistake my styles get compiled.